cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/realtime/source/KafkaSource.java

```diff
@@ -74,12 +85,24 @@ public KafkaSource (KafkaPluginConfig config) {
     this.config = config;
   }
 
+  public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
```

Comment:
Is this overriding method from parent? If it is could you add Override annotation?